docs/nx-cloud/features/split-e2e-tasks.md

```diff
@@ -33,15 +33,301 @@ If you are already using the `@nx/cypress` or `@nx/playwright` plugin, you need
+    "targets.e2e-ci--src/e2e/login.cy.ts.outputs": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci--src/e2e/login.cy.ts.inputs": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci--src/e2e/login.cy.ts.cache": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci--src/e2e/login.cy.ts.options": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci--src/e2e/login.cy.ts.executor": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci--src/e2e/login.cy.ts.options.cwd": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci--src/e2e/login.cy.ts.options.command": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci.executor": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci.cache": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci.inputs": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci.outputs": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e-ci.dependsOn": [
+      "apps/admin-e2e/cypress.config.ts",
+      "@nx/cypress/plugin"
+    ],
+    "targets.e2e.dependsOn": [
+      "apps/admin-e2e/project.json",
+      "nx/core/target-defaults"
+    ],
+    "targets.lint": ["apps/admin-e2e/project.json", "nx/core/project-json"],
+    "targets.lint.executor": [
+      "apps/admin-e2e/project.json",
+      "nx/core/project-json"
+    ],
+    "targets.lint.inputs": [
+      "apps/admin-e2e/project.json",
+      "nx/core/target-defaults"
+    ],
+    "targets.lint.cache": [
+      "apps/admin-e2e/project.json",
+      "nx/core/target-defaults"
+    ],
+    "name": ["apps/admin-e2e/project.json", "nx/core/project-json"],
+    "$schema": ["apps/admin-e2e/project.json", "nx/core/project-json"],
+    "sourceRoot": ["apps/admin-e2e/project.json", "nx/core/project-json"],
+    "tags": ["apps/admin-e2e/project.json", "nx/core/project-json"],
+    "implicitDependencies": [
+      "apps/admin-e2e/project.json",
+      "nx/core/project-json"
+    ],
+    "implicitDependencies.admin": [
+      "apps/admin-e2e/project.json",
+      "nx/core/project-json"
+    ],
+    "targets.lint.outputs": [
+      "apps/admin-e2e/project.json",
+      "nx/core/project-json"
+    ]
+  }
+}
+```
+
+{% /project-details %}
+
 You'll see that there are tasks named `e2e`, `e2e-ci` and a task for each e2e test file.
 
-Developers can run all e2e tests locally the same way as usual:
+Developers can run all e2e tests locally in the standard way:
```

Comment:
should we say what the 'standard way' is here? personally, I think it assumes too much.

Reply:
Perhaps `non-distributed` way is more explicit?